location/src/main/java/org/motechproject/nms/location/service/impl/LocationServiceImpl.java

```diff
@@ -9,4 +11,20 @@
 @Service("locationService")
 public class LocationServiceImpl implements LocationService {
 
+    public static final int HACKY_LANGUAGE_CODE = 34;
+    public static final long HACKY_WHITELISTED_STATE_CODE = -1L;
+    public static final long HACKY_NON_WHITELISTED_STATE_CODE = -10L;
+
+    @Override
+    public State getStateForLanguage(Language language) {
+        // TODO: #71 For a given languageLocationCode find the districts it maps to and find the state for
```

Comment:
I thought that the whole point of language location code was that it would only map to one unique district? (vs. a language code that could map to districts across multiple states?)